test/kubemark/start-kubemark.sh

```diff
@@ -402,21 +446,31 @@ fi
 
 # master machine name format: {KUBE_GCE_ZONE}-kubemark-tp-1-master
 # destination file " --resource-providers=/etc/srv/kubernetes/kube-scheduler/rp-kubeconfig"
+# TODO: do the same for kube-controller-manager for each TP as well
+#
 if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
   for (( tp_num=1; tp_num<=${SCALEOUT_TP_COUNT}; tp_num++ ))
   do
     tp_vm="${RUN_PREFIX}-kubemark-tp-${tp_num}-master"
     echo "DBG: reset scheduler on TP master: ${tp_vm}"
-    gcloud compute scp --zone="${KUBE_GCE_ZONE}" "${RP_KUBECONFIG}" "${tp_vm}:/tmp/rp-kubeconfig"
+    for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
+    do
+      rp_kubeconfig="${RP_KUBECONFIG}-${rp_num}"
+      gcloud compute scp --zone="${KUBE_GCE_ZONE}" "${rp_kubeconfig}" "${tp_vm}:/tmp/rp-kubeconfig-${rp_num}"
+
+      cmd="sudo mv /tmp/rp-kubeconfig-${rp_num} /etc/srv/kubernetes/kube-scheduler/rp-kubeconfig-${rp_num}"
+      gcloud compute ssh --ssh-flag="-o LogLevel=quiet" --ssh-flag="-o ConnectTimeout=30" --project "${PROJECT}" --zone="${KUBE_GCE_ZONE}" "${tp_vm}" --command "${cmd}"
+    done
```

Comment:
should we use custom meta data to transfer rp-kubeconfig file?  gcloud compute ssh/scp are not stable for this purpose.

Reply:
for the TP scheduler and controllers ( in next PR ), scp is used here since the VM is already created. The meta-data in GCE hooks with the VM cloud-init scripts( and flows ) -- such as the kubeconfigs or others that are required to start the k8s master process -- to start the k8s master WHEN the VM is orchestrated.